pkg/apiserver/domain/service/definition.go

```diff
@@ -63,12 +65,13 @@ type definitionServiceImpl struct {
 type DefinitionQueryOption struct {
 	Type             string `json:"type"`
 	AppliedWorkloads string `json:"appliedWorkloads"`
+	OwnerAddon       string `json:"sourceAddon"`
 	QueryAll         bool   `json:"queryAll"`
 }
 
 // String return cache key string
 func (d DefinitionQueryOption) String() string {
-	return fmt.Sprintf("type:%s/appliedWorkloads:%s/queryAll:%v", d.Type, d.AppliedWorkloads, d.QueryAll)
+	return fmt.Sprintf("type:%s/appliedWorkloads:%s/ownerAddon:%s/queryAll:%v", d.Type, d.AppliedWorkloads, d.OwnerAddon, d.QueryAll)
```

Comment:
will it break the compatibility before?

Reply:
\cc @barnettZQG @wangyikewxgm

Reply:
AFAIK, it is only used in caches (when I wrote this code 16 days ago). Now, the cache is removed #4147. So this function is not even used (from my initial inspections).

Reply:
Great